painters/data_dirs_organizer.py

```diff
@@ -29,7 +29,7 @@ def _organize_train_test_dir():
 def _organize_train_dir():
     paths, labels = _load_paths_labels_from_train_dir()
     ind_tr, ind_val, classes = _train_val_split_indices(labels)
-    _save_images_to_dir(NEW_TRAIN_DIR, paths[ind_tr], labels[ind_tr], classes)
+    _save_images_to_dir(NEW_TRAIN_DIR, paths[ind_tr], labels[ind_tr], classes, multi_crop=True)
```

Comment:
What is the motivation behind adding this argument to the function call?

Reply:
to make the code easier to read: the different between this line and the next is the difference between training (with multiple crops of an image created) and validation data (without multiple crops of image data), and I thought writing it explicitly makes it easier to read. Do you think this is a good practice?

Reply:
I see -- yes, that makes sense to me. Maybe add a quick comment explicitly calling this out